docs/theme_override_home/main.html

```diff
@@ -27,7 +27,7 @@
 {% endblock %}
 {% block announce %}
   <style>.md-announce a,.md-announce a:focus,.md-announce a:hover{color:currentColor}.md-announce strong{white-space:nowrap}.md-announce .twitter{margin-left:.2em;color:#00acee}</style>
-    <a href="https://up42.github.io/up42-py/CHANGELOG/"><strong>New version 0.11:</strong> Adds map quicklooks feature & more, see the 📅
+    <a href="https://up42.github.io/up42-py/CHANGELOG/"><strong>New version 0.11.1:</strong> Improves object printouts, adds info & status properties, fixes & docs improvements! See the 📅
```

Comment:
Maybe we can say "Object representation"? Don't understand the last part, "fixes & docs improvements"

Reply:
yup, edited & removed last part.